browser/ui/views/brave_vpn/brave_vpn_fallback_dialog_view.cc

```diff
@@ -82,46 +79,24 @@ BraveVpnFallbackDialogView::BraveVpnFallbackDialogView(Browser* browser)
   const std::u16string contents_text =
       l10n_util::GetStringUTF16(IDS_BRAVE_VPN_FALLBACK_DIALOG_TEXT);
 
-  std::u16string learn_more_link_text =
-      l10n_util::GetStringUTF16(IDS_BRAVE_VPN_FALLBACK_DIALOG_LEARN_MORE_TEXT);
-  std::u16string full_text = l10n_util::GetStringFUTF16(
-      IDS_BRAVE_VPN_FALLBACK_DIALOG_TEXT, learn_more_link_text);
-  const int main_message_length =
-      full_text.size() - learn_more_link_text.size();
-
   auto* contents_label = AddChildView(std::make_unique<views::StyledLabel>());
   contents_label->SetTextContext(views::style::CONTEXT_DIALOG_BODY_TEXT);
   views::StyledLabel::RangeStyleInfo message_style;
-  contents_label->SetText(full_text);
-  contents_label->AddStyleRange(gfx::Range(0, main_message_length),
+  contents_label->SetText(contents_text);
+  contents_label->AddStyleRange(gfx::Range(0, contents_text.size()),
```

Comment:
I think we could delete this style setting as it just sets default.

Reply:
Removed! 😄